README.md

```diff
@@ -172,7 +172,7 @@ Are you using a different runner? A PR would be appreciated 😊.
 
 # Examples
 We are using the API provided by the bundle module 
-[atrium-fluent](https://github.com/robstoll/atrium/tree/main/bundles/fluent/atrium-fluent/build.gradle)
+[atrium-fluent](https://github.com/robstoll/atrium/blob/main/bundles/fluent/atrium-fluent/build.gradle.kts)
```

Comment:
can you keep tree please, although blob makes more sense in this case. we have a rewrite in place which only works with tree currebtly